.github/actions/check-nextjs-bundle-size/action.yaml

```diff
@@ -0,0 +1,83 @@
+name: Check NextJS bundle size
+description: Check NextJS bundle size
+
+inputs:
+  path_to_nextjs_folder:
+    description: 'The path to reach the NextJS folder (e.g. ./apps/website)'
+    required: true
+  upload_path:
+    description: 'The path where the report should be uploaded'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    # Here's the first place where next-bundle-analysis' own script is used
+    # This step pulls the raw bundle stats for the current bundle
+    - name: Generate bundle report
+      working-directory: ${{ inputs.path_to_nextjs_folder }}
+      shell: bash
+      run: npx -p nextjs-bundle-analysis@0.5.0 report
+
+    - name: Upload bundle
+      uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
+      with:
+        name: bundle
+        path: ${{ inputs.upload_path }}
+
+    - name: Download base branch bundle stats
+      id: base-branch-stats
+      uses: dawidd6/action-download-artifact@246dbf436b23d7c49e21a7ab8204ca9ecd1fe615 # v2.27.0
```

Comment:
```suggestion
      id: base-branch-stats
      # Use this action instead actions/download-artifact because the official
      # one doesn't allow to download an artifact from different workflow
      uses: dawidd6/action-download-artifact@246dbf436b23d7c49e21a7ab8204ca9ecd1fe615 # v2.27.0
```